api/test/FakeJigsaw.js

```diff
@@ -69,6 +69,59 @@ app.get('/customer/api/customerSearch', (req, res) => {
   }
 });
 
+app.get('/homelessness/api/casecheck/:id', (req, res) => {
```

Comment:
Not suggesting we change now, but if you're interested take a look at [nock](https://www.npmjs.com/package/nock#usage), you can use it to mock specific endpoints in your tests - so you don't need to spin up a test server. You can also use it to simulate network timeouts etc. if you want to test error scenarios.